src/Aquifer.Common/Jobs/MessagesJsonSerializer.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Newtonsoft.Json;
```

Comment:
Newtonsoft.Json correctly serializes and then deserialized anonymous types into an `object`.  System.Text.Json fails at this.  After many workarounds @jwinston-bn convinced me to just use Newtonsoft.Json here.